### ISSUE TITLE
reddit.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#1042]
### Basic Information

**Test link:** `https://www.reddit.com/r/gtaonline/submit?selftext=true`
**Category:** breakage
**Reported from:** Egypt
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0
**Extension:** Nano Defender 15.0.0.132

### Additional Message

the guiding text that is supposed to go away once you write something in the boxes returns once you cluck outside the box making it difficult to review what you wrote before submitting

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.reddit.com/r/explainlikeimfive/comments/3nf974/eli5_why_do_reddit_posts_get_archived/`
**Category:** ads
**Reported from:** Canada
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36
**Extension:** Nano Defender 15.0.0.133

### Additional Message

ads placeholders on archived posts

**Comment by jspenguin2017** [comment]
I can reproduce neither issues above.

**Comment by jspenguin2017** [comment]
Actually, the new Reddit shows a small ads placeholder in the sidebar. 
![image](https://user-images.githubusercontent.com/7283682/59154776-a1914f80-8a37-11e9-979e-d6ee20a5bc57.png)

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.reddit.com/`
**Category:** anti-adblock
**Reported from:** Australia
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36
**Extension:** Nano Defender 15.0.0.154

### Additional Message

**Comment by jspenguin2017** [comment]
Can't reproduce anti-adblock.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.reddit.com/r/firefox/comments/edu5oa/why_isnt_firefox_on_amazon_fire_tablet_app_store/`
**Category:** ads
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:71.0) Gecko/20100101 Firefox/71.0
**Extension:** Nano Adblocker 1.0.0.131

### Additional Message

leftovers ads:

`##div[id*="sidebar"][data-before-content="advertisement"]:nth-ancestor(4)`
